Sources/BraintreeThreeDSecure/BTThreeDSecureClient.swift

```diff
@@ -88,17 +90,20 @@ import BraintreeCore
         self.request = request
         
         guard apiClient.clientToken != nil else {
+            self.apiClient.sendAnalyticsEvent(BTThreeDSecureAnalytics.verifyFailed)
```

Comment:
I wasn't sure about the flow in the case merchant calls this function directly. In the flow where the private function `prepareLookup` gets called form `startPaymentflow` function, we send `verifyFailed` analytic messages from completion handler of the private` prepareLookup` function.

Reply:
Yes, since this guard calls `completion(nil, BTThreeDSecureError.configuration("A client token must be used for ThreeDSecure integrations."))` right after this we can handle this the same way we handle any other completion with an error in this file.